packages/components/src/inputs/src/CharacterCount.tsx

```diff
@@ -0,0 +1,30 @@
+import { useLocalizedString } from "../../i18n/index.ts";
+import { Text } from "../../typography/index.ts";
+
+import styles from "./CharacterCount.module.css";
+
+export interface CharacterCountProps {
+    charactersLeft: number;
+    isDisabled?: boolean;
+    isInvalid?: boolean;
+}
+
+export function CharacterCount({ charactersLeft, isDisabled, isInvalid, ...props }: CharacterCountProps) {
```

Comment:
I have a comment below about the tests. One of the tests includes checking the ref, which means we should probably add the same stuff here as the other components for ref as well as the styled system stuff.
Users will be able to use this component if they really wanted to.

Reply:
Would that component be discoverable? If there's no documentation for it? Should there be?

Reply:
Yeah I'm not sure if the goal of this ticket was to make this discoverable or just to be used internally